src/main/java/org/folio/edge/rtac/utils/RtacMimeTypeEnum.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.edge.rtac.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.folio.edge.core.Constants;
+
+
+public enum RtacMimeTypeEnum {
```

Comment:
This enum doesn't seem to provide any behaviour. What benefit does this enum have over the constants themselves?

Reply:
There were several causes why I introduced enum. Of course we can re-use constants from edge-common, but not all values are listed there, I mean "all" value, and we can't made changes there. Another point is that enum can bare some of functionality like simplify providing all the types or xml-related ones via one call instead of just listing all of strings. And the last one - this approach is more meaningfull from semantic point of view in comparison with just strings as for me, which are error prone by their nature. But if you think that enum brings extra code here, I can duplicate constant-types here in this module too.

Reply:
> There were several causes why I introduced enum. Of course we can re-use constants from edge-common, but not all values are listed there, I mean "all" value, and we can't made changes there. 

> Another point is that enum can bare some of functionality like simplify providing all the types or xml-related ones via one call instead of just listing all of strings. 

As far as I can tell, the enumeration has no behaviour associated with it, and does not do that. Have I missed something?

> this approach is more meaningful from semantic point of view in comparison with just strings as for me, which are error prone by their nature. 

I agree with that general intent. To me, the enum as it currently is, is effectively only a grouping of constants, so I didn't understand what value it had over the constants. Especially as the behaviour around the media types is defined in a separate utility class.

> But if you think that enum brings extra code here, I can duplicate constant-types here in this module too.

I'm not suggesting duplicating any constants.

Reply:
Yes Marc I understand what you mean in general. To be honest I personally have a feeling that this enum serve only constants container purposes here, because rest of "behavior" can be simply replaced by reusing of enum entries itself. Further extension of this enum type of supported rtac request types isn't implied. Therefore only one purpose of such approach is obvious - constants container. If this reason can excuse of enum-style approach, I'll leave it as is, but just slightly simplify. If it would be better to avoid extra unneeded types in code, I think entries from edge-common constants can be used instead with only one constant added in utility class, I mean wildcard "all". The last approach will be less code in result.

Reply:
> I think entries from edge-common constants can be used instead with only one constant added in utility class, I mean wildcard "all". The last approach will be less code in result.

I agree with this conclusion.

Let's not change this further at this point in the interests of the limited time the team have for work for 2021 R1.